joomla/joomla-cms
# Put back Loading/ok/error images while changing permission [solve regression]

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/12843.

### Summary of Changes

In conjunction with https://github.com/joomla/joomla-cms/pull/12855 (that takes care of the search in the permissions select box), this PR corrects the CSS so the ok/error/loading image is shown again as in 3.6.4.

### Testing Instructions

Very simple test. Just CSS change.
- Go to any permissions tab and change any permission. You will notice there is no image showing loading/success/error.
- Apply patch and clear browser cache
- Go to any permissions tab and change any permission, you will now the image appears now and also the width of the columns stay stable (this is an improvement comparing to 3.6.4).
![image](https://cloud.githubusercontent.com/assets/9630530/20237518/7b16dc16-a8cc-11e6-934c-951c82702b83.png)

### Documentation Changes Required

None.
